requirements.txt

```diff
@@ -1,2 +1,3 @@
 ruamel.yaml
-auth0-python
\ No newline at end of file
+auth0-python
+git+https://github.com/GeorgianaElena/jupyterhub_client.git@user-token
```

Comment:
This is pinned to a branch (`user-token`). Instead, can you pin to a git commit hash? That way, we know exactly what's installed. And we can bump it when your PR gets merged, or when you add a new commit.